app/controllers/avo/associations_controller.rb

```diff
@@ -190,5 +194,36 @@ def has_many_reflection?
         ActiveRecord::Reflection::HasAndBelongsToManyReflection
       ]
     end
+
+    def through_reflection?
+      @reflection.instance_of? ActiveRecord::Reflection::ThroughReflection
+    end
+
+    def additional_params
+      params[:fields].permit(@attach_fields.map(&:id))
```

Comment:
One more thing.
Let's see if this `permit` statement emits a red warning in the console for other params which are sent by the modal.

I'd like to prevent that behavior. Some users when they have a problem with that actions, they will look at that and think that that is the cause of the problem.
So they are led astray.